tfprotov5/client_capabilities.go

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package tfprotov5
+
+// ClientCapabilities allows Terraform to publish information regarding optionally supported
+// protocol features, such as forward-compatible Terraform behavior changes.
+type ClientCapabilities struct {
+	// DeferralAllowed signals that the Terraform client is able to
```

Comment:
**Reminder from offline convo:** Update these pkg docs to reflect that the client capability is related to the specific request and not necessarily the client

Reply:
Fixed pkg comments in 3a4fbc8 ⚡